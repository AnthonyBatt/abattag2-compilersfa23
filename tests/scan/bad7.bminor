"Boolean logic still isn't supported"
|
