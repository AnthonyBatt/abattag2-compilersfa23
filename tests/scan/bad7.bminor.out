Program exiting, there was a scanning error
TOKEN_STRING_LITERAL      Boolean logic still isn't supported
There was a scan error on |
